numpy/_core/tests/test_shape_base.py

```diff
@@ -379,7 +381,8 @@ def test_concatenate(self):
         assert_(out is rout)
         assert_equal(res, rout)
 
-    @pytest.mark.skipif(IS_PYPY, reason="PYPY handles sq_concat, nb_add differently than cpython")
+    @pytest.mark.skipif(IS_PYPY,
+                    reason="PYPY handles sq_concat, nb_add differently than cpython")
```

Comment:
```suggestion
    @pytest.mark.skipif(
        IS_PYPY,
        reason="PYPY handles sq_concat, nb_add differently than cpython"
    )
```

or

```suggestion
    @pytest.mark.skipif(IS_PYPY,
                        reason="PYPY handles sq_concat, nb_add differently than cpython")
```

(if that even fits)